addon-test-support/utils/find-tooltip.js

```diff
@@ -11,7 +11,12 @@ export default function findTooltip(selector) {
   children of <body> instead of children of the $targetElement */
 
   const $body = $(document.body);
-  const $tooltip = $body.find(selector);
+  let $tooltip = $body.find(selector);
+
+  if ($tooltip.length && $tooltip.hasClass('ember-tooltip-base')) {
```

Comment:
Probably worth documenting why this `if` statement is necessary.

Reply:
good call. will do 👍